File: fms2/src/main/java/com/fms2/fms2/receivableAndPayable/domain/PayableStatistics.java
package com.fms2.fms2.receivableAndPayable.domain;

public class PayableStatistics {
    private String supplier;
    private String startDate;
    private String endDate;
    private Double purchaseAmount;
    private Double actualPayment;
    private Double balance;

    public String getSupplier() {
        return supplier;
    }

    public void setSupplier(String supplier) {
        this.supplier = supplier;
    }

    public String getStartDate() {
        return startDate;
    }

    public void setStartDate(String startDate) {
        this.startDate = startDate;
    }

    public String getEndDate() {
        return endDate;
    }

    public void setEndDate(String endDate) {
        this.endDate = endDate;
    }

    public Double getPurchaseAmount() {
        return purchaseAmount;
    }

    public void setPurchaseAmount(Double purchaseAmount) {
        this.purchaseAmount = purchaseAmount;
    }

    public Double getActualPayment() {
        return actualPayment;
    }

    public void setActualPayment(Double actualPayment) {
        this.actualPayment = actualPayment;
    }

    public Double getBalance() {
        return balance;
    }

    public void setBalance(Double balance) {
      this.balance = balance;
    }
}

File: fms2/src/main/resources/static/Myapp/app/model/PayableStatistics.js
Ext.define('MyApp.model.PayableStatistics', {
    extend: 'MyApp.model.Base',

    fields: [
        {name:'supplier',type: 'string'},
        {name:'startDate',type:'string'},
        {name:'endDate',type:'string'},
        {name:'purchaseAmount',type:'string'},
        {name:'actualPayment',type:'string'},
        {name:'balance',type:'string'},
    ],

});
